tests/rptest/services/redpanda_cloud.py

```diff
@@ -1306,6 +1358,58 @@ def _create_vpc_peering_fmc(self):
             # that was created already above
             self.current.vpc_peering_id = self.provider_cli.create_vpc_peering(
                 self.current, facing_vpcs=False)
+        elif self.config.provider == PROVIDER_AZURE:
+            # TODO This part was not yet tested. Azure cloud cluster creation for Tier testing is blocked. Added this section based on AWS, but might need to update it
```

Comment:
This part was not yet tested, since we are blocked on testing and can't even create Cloud clusters on Azure. Might need some updates once testing gets unblocked